image/image.go

```diff
@@ -223,6 +223,17 @@ func (i Ref) WithNewTag(t string) Ref {
 	return img
 }
 
+type LabelParseError struct {
```

Comment:
should we make the error more explicit? e.g. `TimestampFormatError`

Reply:
`LabelTimestampFormatError` even maybe?

Reply:
Sure!